tests/adapter/dbt/tests/adapter/aliases/test_aliases.py

```diff
@@ -0,0 +1,354 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+
+MODELS_DUPE_CUSTOM_DATABASE__SCHEMA_YML = """
```

Comment:
future nit: might be easier to read if it's in a separate fixtures.py

Reply:
We decided as a team if there are more than 50 lines of these, they should be defined in fixtures.py and then import the strings.  We definitely don't do this everywhere right now but should move to this model.

Example: [test_seed.py](https://github.com/dbt-labs/dbt-core/blob/main/tests/functional/simple_seed/test_seed.py)